<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_clustering_config</class>
 <widget class="QWidget" name="widget_clustering_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lexical diversity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_lexical_diversity">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Percentage of stopwords:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_percentage_stopwords">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>761</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox { 
	background-color: rgb(234, 234, 234);
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
   </property>
   <property name="title">
    <string>Feature extraction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QCheckBox" name="check_perform_lsa_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>perform lsa</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>number of components:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_n_components_clustering_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>explained variance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_explained_variance_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>59</y>
      <width>192</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ngram range:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_ngram_range_clustering_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>max df:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_max_df_clustering_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>min df:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_min_df_clustering_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>max features: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_max_features_clustering_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_vectorize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>vectorize data</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>59</y>
      <width>195</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout_checkboxes_2">
     <item>
      <widget class="QCheckBox" name="check_use_stemming_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>use stemming</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_use_idf_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>use idf</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_binary_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>binary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_remove_stopwords_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>remove stopwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>vectorizer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_vectorizer_clustering_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>209</y>
      <width>219</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>features:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_features_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox_2 { 
	 background-color: rgb(234, 234, 234);
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
   </property>
   <property name="title">
    <string>K-means</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>96</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="0,0">
     <item>
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>k:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_clusters_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="check_use_minibatch_clustering">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>193</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>use minibatch k-means</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clusterize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>clusterize data</string>
    </property>
    <property name="text">
     <string>clusterize</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>136</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <widget class="QLabel" name="label_24">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>silhuette coeficient:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_silhuette_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>110</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>done in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_donein_clustering_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Twitter dataset:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_open_file">
      <property name="toolTip">
       <string>open</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>53</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_save_clustering_config">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>500</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;save clusterized data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_save_clustering_config {

	background-color: transparent;	
	background-image: url(:/save/save.png);
	background-repeat: no-repeat;
}


QPushButton#button_save_clustering_config:hover {
	background-color: transparent;
	
	background-image: url(:/save_2/save_2.png);
	
	background-repeat: no-repeat;
}
 
QPushButton#button_save_clustering_config:pressed {
	background-color: transparent;
	
	background-image: url(:/save/save.png);
	
	background-repeat: no-repeat;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>33</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/save.qrc"/>
  <include location="../assets/save_2.qrc"/>
 </resources>
 <connections/>
</ui>
